internal/fuzzbinder/util.go

```diff
@@ -0,0 +1,43 @@
+// Copyright 2022 The Project Oak Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package fuzzbinder
+
+// This file provides utility functions for FuzzBinder tool.
+
+import (
+	"fmt"
+	"time"
+)
+
+// ValidateDate validates that the fuzzing date chosen to generate the fuzzing
+// claims is no more than 15 days prior to the date of execution of FuzzBinder cmd
+// and not in the future.
+func ValidateDate(date string) error {
+	claimDate, err := time.Parse("20060102", date)
+	if err != nil {
+		return fmt.Errorf(
+			"the format of %s is not valid: the date format should be yyyymmdd", date)
+	}
+	currentTime := time.Now()
+	duration := currentTime.Sub(claimDate).Hours() / 24
```

Comment:
I think that this computation does depend on when / where the tool is called.

Can we think of a better solution? When does a day start in the logs, i.e., which time zone? Can we go with that time zone?

However, maybe it is just sufficient to make sure that logs are actually---because that is what really matters to us? What do you think?

Reply:
Another error will be returned if the logs are not found anyway (so this lack of precision will not cause a problem for FuzzBinder). However, I agree with the fact that this verification will depend on the time zone for the day -15 and current day.  

I will look at the exact time zone that is used for saving logs. However, I did not do it before because I am not sure if the  `ossFuzzLogRetentionDays`  is equal to 15.00 or about 15 (because it was an observation). For more precision, I will look if I can have access to both.

Reply:
I agree, it would be more accurate to use the correct timezone. I am also fine with allowing 16 days and failing if there are any logs (I looked around a bit and it is not clear to me which time zone is used for the logs).

You can also shorten the code by using `IsAfter`/`IsBefore` and `AddDate(0, 0, -15)`.

Reply:
I think the time zone is UTC.

Reply:
I updated the time-zone and used `After/Before` to shorten the code.